### PR TITLE
fix: add contractAddress for upload service

### PR DESCRIPTION
**Pull request by julianmrodri**
* Fixes the upload service providing the additional parameter. The `storageManager` contract address.

### CODE REVIEW

File: src/context/Services/storage/upload/Context.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import React, {
 } from 'react'
 import { createReducer } from 'context/storeUtils/reducer'
 import AppContext, { AppContextProps, errorReporterFactory } from 'context/App/AppContext'
+import networkConfig from 'config'
```
Comment (farinalvaro) [inline src/context/Services/storage/upload/Context.tsx:7]: We could directly import the storage contract address here

**Review by farinalvaro — COMMENTED** [review]
Looks good. Just that small comment

File: src/context/Services/storage/upload/Context.tsx (hunk 1 above)
Comment (julianmrodri) [reply src/context/Services/storage/upload/Context.tsx:7]: Sounds good.

**Review by farinalvaro — APPROVED** [review]
LGTM

**Pull request merged**